configure.ac

```diff
@@ -313,6 +321,9 @@ AC_CONFIG_LINKS([ \
   t/fluxometer.lua:t/fluxometer.lua \
 ])
 
+AC_CONFIG_FILES([etc/rc1], [chmod +x etc/rc1])
+AC_CONFIG_FILES([etc/rc3], [chmod +x etc/rc3])
+
 AC_OUTPUT
```

Comment:
Optional, but it would be nice if fixes like this were split out from the jobspec related changes.

Reply:
They are in a separate commit.  I'll just drop the job ingest stuff from this PR like Jim suggested.